src/views/Unsubscribe.vue

```diff
@@ -1,148 +1,81 @@
 <script setup lang="ts">
-import { ref, computed, onMounted, Ref } from 'vue';
+import { ref, onMounted } from 'vue';
 import { useRoute } from 'vue-router';
 import { useSeoMeta } from '@vueuse/head';
 import BasePage from '@/components/BasePage.vue';
-import LoadingSpinner from '@/components/LoadingSpinner.vue';
 import MessageBody from '@/components/MessageBody.vue';
-import MessageBox from '@/components/MessageBox.vue';
-import InputCheckbox from '@/components/InputCheckbox.vue';
 import RedirectButton from '@/components/RedirectButton.vue';
 import BaseButton from '@/components/BaseButton.vue';
 
 enum Status {
   UNKNOWN,
-  WAITING,
   SUCCESS,
   ERROR
 }
 
 let status = ref(Status.UNKNOWN);
 let loading = ref(false);
-let subscriptionsList: Ref<Record<string, Record<string, string>>> = ref({});
-let updatedSubscriptions: Ref<string[]> = ref([]);
 const route = useRoute();
 
 useSeoMeta({
-  title: 'Manage email subscriptions',
-  description: 'Manage your snapshot mailing subscriptions'
+  title: 'Unsubscribe',
+  description: 'Unsubscribe from Snapshot mailing list'
 });
 
-async function initForm() {
-  try {
-    const response = await fetch(`${import.meta.env.VITE_API_URL}/subscriptionsList`);
-    status.value = Status.WAITING;
-
-    if (response.status === 200) {
-      subscriptionsList.value = await response.json();
-      updatedSubscriptions.value = Object.keys(subscriptionsList.value);
-    } else {
-      status.value = Status.ERROR;
-    }
-  } catch (error) {
-    status.value = Status.ERROR;
-    loading.value = false;
-    console.log(error);
-  }
-}
-
-function update() {
-  return unsubscribe(updatedSubscriptions.value);
-}
-
-function unsubscribeFromAll() {
-  return unsubscribe([]);
-}
-
-async function unsubscribe(subscriptions: string[]) {
+function unsubscribe() {
   loading.value = true;
 
-  try {
-    const response = await fetch(`${import.meta.env.VITE_API_URL}`, {
-      method: 'POST',
-      headers: {
-        'Content-Type': 'application/json'
+  fetch(`${import.meta.env.VITE_API_URL}`, {
```

Comment:
also you don't need interpolation here
```suggestion
  fetch(import.meta.env.VITE_API_URL, {
```